src/patternfly/components/Table/table.scss

```diff
@@ -499,6 +512,15 @@
       --pf-c-table-cell--PaddingRight: var(--pf-c-table--nested--first-last-child--PaddingRight);
     }
   }
+
+  .pf-c-table__expandable-row {
+    .pf-c-table__expandable-row-content {
+      --pf-c-table__expandable-row-content--PaddingTop: var(--pf-c-table--m-compact__expandable-row-content--PaddingTop);
```

Comment:
The paddings on all expandable-row-contents should be 24px. 

I had updated table-grid.scss to this, but for some reason the compact table is not assigning it to the only child example (second one).
 `tr > * {
    // Remove First child padding left
    &:first-child {
      --pf-c-table-cell--PaddingLeft: 0;
    }
    // Remove Last child padding right
    &:last-child {
      --pf-c-table-cell--PaddingRight: 0;
    }
  }
`
<img width="528" alt="Screen Shot 2019-03-28 at 11 15 51 AM" src="https://user-images.githubusercontent.com/20118816/55169277-e1d87580-514a-11e9-95f8-11e9d693a8ed.png">

Reply:
ah! Thanks. Updated, let me know if that addresses it :)